src/sentry/integrations/example/integration.py

```diff
@@ -65,7 +65,8 @@ def get_issue(self, issue_id):
     def sync_assignee_outbound(self, external_issue, user, assign=True, **kwargs):
         pass
 
-    def sync_status_outbound(external_issue, is_resolved):
+    def sync_status_outbound(external_issue, is_resolved, project_id):
+        # TODO(lb): should this have self?
```

Comment:
Based on the actual class, I think so, yea.

Reply:
Cool. I think jess just might have been in a hurry. :)